e2e-tests/secure_supply_chain_tests.bats

```diff
@@ -1,10 +1,24 @@
 #!/usr/bin/env bats
 
-FULCIO_AND_REKORD_DATA_DIR=$(readlink -f ~/.config/kubewarden/fulcio_and_rekor_data)
+xdg() {
```

Comment:
Suggestion: rename to a more descriptive name: `setup_xdg_environment` or something along the lines.

Reply:
done in last commit, https://github.com/kubewarden/kwctl/pull/183/commits/775dc214a8cd5bf5e847c9700316583976532a6f